Dockerfile

```diff
@@ -0,0 +1,69 @@
+## Build
+FROM golang:1.22-alpine AS build
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY go.sum ./
+RUN go mod download
+
+COPY *.go ./
+ADD constants /app/constants
+ADD github /app/github
+
+RUN go build -o /raito-cli
+
+## Deploy
+FROM alpine:3 as deploy
+
+LABEL org.opencontainers.image.base.name="alpine:3"
+
+RUN apk add --no-cache tzdata
+
+WORKDIR /app
+
+RUN mkdir -p /config
+
+ENV TZ=Etc/UTC
+ENV CLI_CRON="0 2 * * *"
+ENV RAITO_CLI_UPDATE_CRON="0 1 * * *"
+ENV RAITO_CLI_CONTAINER_STDOUT_FILE="/dev/stdout"
+ENV RAITO_CLI_CONTAINER_STDERR_FILE="/dev/stderr"
+
+RUN addgroup -S raito && adduser -D -S -G raito --no-create-home raito && chmod +w /tmp
+RUN chown raito:raito /app /config
+
+COPY --from=build /raito-cli /raito
+RUN chown raito:raito /raito
+
+USER raito
+
+ENTRYPOINT /raito run -c "$CLI_CRON" --config-file /config/raito.yml --log-output
+
+## Deploy-amazon
+FROM amazon/aws-cli:2.15.10 as amazonlinux
```

Comment:
Would update to the latest version --> [2.15.31](https://hub.docker.com/r/amazon/aws-cli/tags)